aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -71,56 +46,90 @@ The following minimum changes are required to set up an existing ASP.NET Core pr
 
    * Call [UseContentRoot](xref:fundamentals/host/web-host#content-root) and use a path to the app's published location instead of `Directory.GetCurrentDirectory()`.
 
-     ::: moniker range=">= aspnetcore-2.0"
-
      [!code-csharp[](windows-service/samples/2.x/AspNetCoreService/Program.cs?name=ServiceOnly&highlight=8-9,16)]
 
-     ::: moniker-end
+1. Publish the app. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles). When using a Visual Studio, select the **FolderProfile** and configure the **Target Location** before selecting the **Publish** button.
 
-     ::: moniker range="< aspnetcore-2.0"
+   To publish the sample app using command-line interface (CLI) tools, run the [dotnet publish](/dotnet/core/tools/dotnet-publish) command at a command prompt from the project folder. The RID must be specified in the `<RuntimeIdenfifier>` (or `<RuntimeIdentifiers>`) property of the project file. In the following example, the app is published in Release configuration for the `win7-x64` runtime to a folder created at *c:\\svc*:
 
-     [!code-csharp[](windows-service/samples_snapshot/1.x/AspNetCoreService/Program.cs?name=ServiceOnly&highlight=3-4,8,13)]
+   ```console
+   dotnet publish --configuration Release --runtime win7-x64 --output c:\svc
+   ```
 
-     ::: moniker-end
+1. Create a user account for the service using the `net user` command:
 
-1. Publish the app. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles). When using a Visual Studio, select the **FolderProfile**.
+   ```console
+   net user {USER ACCOUNT} {PASSWORD} /add
+   ```
 
-   To publish the sample app using command-line interface (CLI) tools, run the [dotnet publish](/dotnet/core/tools/dotnet-publish) command at a command prompt from the project folder. The RID must be specified in the `<RuntimeIdenfifier>` (or `<RuntimeIdentifiers>`) property of the project file. In the following example, the app is published in Release configuration for the `win7-x64` runtime:
+   For the sample app, create a user account with the name `ServiceUser` and a password (for example, `Passw0rd$`):
```

Comment:
@blowdart 	do we really want to suggest passwords customers will copy/paste?

Reply:
I removed it and tell the user to replace `{PASSWORD}` with the user account's password in these spots.